spyder_kernels/console/kernel.py

```diff
@@ -239,7 +239,7 @@ def copy_value(self, orig_name, new_name):
         ns = self._get_reference_namespace(orig_name)
         ns[new_name] = ns[orig_name]
 
-    def load_data(self, filename, ext):
+    def load_data(self, filename, ext, override=False):
         """Load data from filename"""
```

Comment:
Please add a period at the end of the string. Also I think it is a good opportunity to explain what override  does and does not do.